tilavarauspalvelu/management/commands/data_creation/create_reservations.py

```diff
@@ -269,7 +268,7 @@ def _create_normal_reservations(
 
 def _build_payment_order(reservation: Reservation, payment_types_choices: list[PaymentType]) -> PaymentOrder:
     payment_order_builder = PaymentOrderBuilder().set(
-        language=reservation.reservee_language or Language.FI,
+        language=reservation.user.get_preferred_language(),
```

Comment:
`reservation.user` is nullable, so accessing the `user` might be an issue in some cases

This comment applies for other changes as well

Reply:
In practice this is not true since the user is always set for all reservations and only removed if the actual user is removed from the system (has `on_delete=SET_NULL`). I also think operations on reservations should fail if there is no user for it.